resources-path-fix.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/setup.py b/setup.py
+index 51083f9..2f622ea 100755
+--- a/setup.py
++++ b/setup.py
+@@ -149,11 +149,11 @@ DESCRIPTION = 'Persepolis Download Manager'
+ 
+ if os_type == 'Linux':
+     DATA_FILES = [
+-        ('/usr/share/man/man1/', ['man/persepolis.1.gz']),
+-        ('/usr/share/applications/', ['xdg/com.github.persepolisdm.persepolis.desktop']),
+-        ('/usr/share/metainfo/', ['xdg/com.github.persepolisdm.persepolis.appdata.xml']),
+-        ('/usr/share/icons/hicolor/scalable/apps/', ['resources/persepolis.svg']),
+-        ('/usr/share/icons/hicolor/scalable/apps/', ['resources/persepolis-tray.svg'])
++        ('/app/share/man/man1/', ['man/persepolis.1.gz']),
```

Comment:
You should submit a patch so that the upstream app honour the PREFIX, like it should.

Reply:
OK, Will submit them a PR regarding prefix and appdata.

Reply:
@hfiguiere Can you tell me how will I get the prefix argument inside python, I tried my best but can't find how it will happen.